src/en/users-creating.md

```diff
@@ -83,8 +83,11 @@ juju login
 ```
 
 Once a user logs in they become the current user. The following is a quick way
-to determine the current user:
+to determine the current user (as well as the current controller and model):
 
 ```bash
-juju show-user
+juju whoami
 ```
+
+The command `juju show-user` can also be used to get information on the current
```

Comment:
Would it be useful to remove the word "also" since `show-user` appears to do more than `whoami`?